packages/core/src/components/Button/Button.stories.tsx

```diff
@@ -28,14 +37,27 @@ export default {
   component: Button,
   decorators: [
     (storyFn: FunctionComponentFactory<{}>) => (
-      <MemoryRouter>
-        <div>
+      <>
+        <Typography>
+          A collection of buttons that should be used in the Backstage
+          interface. These leverage the properties inherited from{' '}
+          <Link href="https://material-ui.com/components/buttons/">
```

Comment:
Do we want to recommend the use of the material-ui link or should we recommend the Link from @backstage/core?

Reply:
Maybe a more general question. If something is a link, do we still want to use a button? For me links mean that I can use browser features like "open in new tab" which is a bit different fro the behavior I expect from a button.

Reply:
The core Link! That's the one we can adorn with things like automatic `_blank` + `noopener noreferrer` for external links, click tracking, etc

Reply:
I wonder whether we should rename our link to something like BLink, BsLink, BSLink... to avoid wrong imports? The same pattern could be used for other components too where we provide a modified variant with the same name.

Reply:
I veto [`<BSLink>`](https://www.urbandictionary.com/define.php?term=BS), due to it's double entrendre. 🤣

Reply:
And note, for the purpose of this PR, the only open we have is the Material-UI `<Link>` as the `@backstage/core` `<Link>` component does not support href's or putting URLs in the `to={}` attribute yet.  When that's done, this could absolutely be refactored!

Reply:
Are you sure that the core link doesn't support absolute `to` links? https://github.com/backstage/backstage/blob/master/packages/core/src/components/Link/Link.tsx#L36
Haven't used it for a while.

Reply:
Yeah it is meant to. The prop is indeed called `to` but that's just semantics

Reply:
I think it could be useful to mention that this Link is just an embryo and we'll want to grow its capabilities. The internal core Link had a lot of features that weren't lifted over yet. So that's an argument to want to spread its use from the start - that way, existing code will benefit from future style/behavior improvements

Reply:
Think we can use the regular link here tbh, but +1 to `Link` needing some work, along with `Button` and a lot of other components 😅